src/main/java/cn/nukkit/block/BlockNetherite.java

```diff
@@ -0,0 +1,56 @@
+package cn.nukkit.block;
+
+import cn.nukkit.item.Item;
+import cn.nukkit.item.ItemTool;
+import cn.nukkit.utils.BlockColor;
+
+public class BlockNetherite extends BlockSolid{
+    public BlockNetherite() {
+    }
+
+    @Override
+    public int getId() {
+        return NETHERITE_BLOCK;
+    }
+
+    @Override
+    public String getName() {
+        return "Netherite Block";
+    }
+
+    @Override
+    public int getToolType() {
+        return ItemTool.TYPE_PICKAXE;
+    }
+
+    @Override
+    public double getHardness() {
+        return 35;
+    }
+
+    @Override
+    public double getResistance() {
+        return 6000;
+    }
+
+    @Override
+    public Item[] getDrops(Item item) {
+        if (item.isPickaxe() && item.getTier() >= ItemTool.TIER_DIAMOND) {
+            return new Item[]{
+                    toItem()
+            };
+        } else {
```

Comment:
there's no need for placing an else here

Reply:
The else was used on this occasion to maintain the pattern followed by the other blocks.